Pipfile

```diff
@@ -11,7 +11,6 @@ disease-normalizer = {version = "~=0.4.0.dev3", extras = ["etl"]}
 thera-py = {version = "~=0.5.0.dev3", extras = ["etl"]}
 civicpy = "~=3.0.0"
 requests = "*"
-jsondiff = "*"
```

Comment:
we can remove this since the delta module is (at least for now) gone right?

Reply:
Yes. Good catch!